apps/microsoft-teams/frontend/src/components/config/AccessSection/AccessSection.spec.tsx

```diff
@@ -1,13 +1,22 @@
 import AccessSection from './AccessSection';
 import { describe, expect, it, vi } from 'vitest';
 import { render, screen } from '@testing-library/react';
+import { accessSection } from '@constants/configCopy';
 
 const tenantIdValue = 'abc-123';
 
 describe('AccessSection component', () => {
   it('mounts with tenantId provided', () => {
-    render(<AccessSection dispatch={vi.fn()} tenantId={tenantIdValue} />);
+    const { unmount } = render(<AccessSection dispatch={vi.fn()} tenantId={tenantIdValue} />);
```

Comment:
Is this for clean up (using the de-structred unmount) for memory leaks, or do we want to assert that these have been unmounted? If thats the case we can do a `.toBeNull()` check on the elements for example:

```typescript
const element= screen.getByText(accessSection.title);
unmount();
expect(element).toBeNull();
```

totally feel free to ignore this as well :D

Reply:
Good callout, this is for cleanup. We learned from AICG that with using `vitest` if you have multiple tests in the same `describe` block, you need to unmount the component between tests. If I remove the unmount, then the tests fail. So I don't think I necessarily need to assert that they have been unmounted, unless you think it would be helpful!